docs/source/concepts/time.rst

```diff
@@ -0,0 +1,90 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _time:
+
+Time
+####
+
+The DAML language contains a function :ref:`getTime <daml-ref-gettime>` which returns the “current time”.
+However, the notion of time comes with a lot of problems in a distributed setting.
+
+This document describes the detailed semantics of time on DAML ledgers,
+centered around the two timestamps assigned to each transaction:
+the *ledger time* ``lt_TX`` and the *record time* ``rt_TX``.
+
+Ledger time
+***********
+
+The *ledger time* ``lt_TX`` is a property of a transaction.
+It is a timestamp with microsecond resolution,
+and defines the value of all :ref:`getTime <daml-ref-gettime>` calls in the given transaction.
+The ledger time is assigned by the participant as part of the DAML command interpretation.
+
+The ledger time of valid transaction ``TX`` must fullfil the following rules:
+
+#. **Causal monotonicity**: for any action (create, exercise, fetch, lookup) in ``TX``
+   on a contract ``C``, ``lt_TX >= lt_C``,
+   where ``lt_C`` is the ledger time of the transaction that created ``C``.
+
+#. **Skew**: ``rt_TX - skew_min <= lt_TX <= rt_TX + skew_max``,
+   where ``skew_min`` and ``skew_max`` are parameters defined by the ledger.
+
+Apart from that, no other guarantees are given on the ledger time.
+Time has therefore to be considered slightly fuzzy in DAML, with the fuzziness depending on the skew parameters.
+DAML applications should not interpret the value returned by :ref:`getTime <daml-ref-gettime>` as a precise timestamp.
+
+
+Record time
+***********
+
+The *record time* ``rt_TX`` is another property of a transaction.
+It is timestamp with microsecond resolution,
+and is assigned by the ledger when the transaction is recorded on the ledger.
+
+The DAML ledger model does not prescribe how the record time is assigned.
```

Comment:
I think we should say something along the lines of it being some intuitive representation of "real time". Then you could also explain that in a distributed system there isn't a single notion of real time, throw in space-time if you feel like it - I'm sure @daravep would approve muchly ;)

I also think we should specifically state that record time needn't increase monotonically (and the user thus shouldn't rely on that), as it might be non-obvious.

Reply:
Done.